modules/misunderstood/src/hooks/after_incoming_middleware/00_misunderstood_flag.js

```diff
@@ -0,0 +1,18 @@
+const axios = require('axios')
+
+const flag = async () => {
+  if (event.type === 'text' && event.nlu.intent.name === 'none') {
```

Comment:
This looks like it will be flagging a ton of messages (anytime the user moves from a node to another one on the flow), since the user is not 100% of the time trying to trigger an intent

Reply:
This is implemented according to my understanding of the requirement `The recognized intent equals “none”, in which case the message will automatically get flagged` and Sylvain's explanation of it. If the condition should be different I need it explained.